server/modules/analysis/service/rChainService/rChain.js

```diff
@@ -162,7 +170,7 @@ class RChain {
   }
 
   async init() {
-    await this._initSurveyAndChain()
+    await this._initSurveyChainAndCategories()
```

Comment:
I would leave the old name, categories are part of the survey, plus you are initializing them here

Reply:
ok